src/jit/codegencommon.cpp

```diff
@@ -10241,6 +10241,62 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 // Look in CodeGenArm64.cpp
 
+#elif defined(_TARGET_X86_)
+
+/*****************************************************************************
+ *
+ *  Generates code for an EH funclet prolog.
+ */
+
```

Comment:
typically, it would be (1) set up PSPSym (if required), (2) save/restore callee-save registers, (3) handle incoming args/return value. I assume you'll at least need to do (2).